recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -228,6 +228,18 @@ def upload_package(feedstock_root, recipe_root, config_file, validate, private,
                     "is not allowed" % ("conda-forge", source_channel))
                 return
 
+    build_tool = determine_build_tool(feedstock_root)
+    if build_tool != CONDA_BUILD and upload_to_conda_forge:
+        # make sure that we are not uploading to the main conda-forge channel
+        # when building packages with `rattler-build`
+        if ["conda-forge", "main"] in channels:
+            print(
+                "Uploading to conda-forge's main channel is not yet allowed when building with rattler-build.\n"
+                "You can set a label channel in the channel_targets section of the config file\n"
+                "to upload to a label channel."
+            )
```

Comment:
Wonder if we should open an issue to track discussion on this test rollout so we can evaluate when we are ready to enable uploads